core/src/main/scala/cats/data/WriterT.scala

```diff
@@ -87,8 +87,7 @@ private[data] sealed abstract class WriterTInstances0 extends WriterTInstances1
       implicit val L0: Monoid[L] = L
     }
 
-  implicit def writerTIdFunctor[L]: Functor[WriterT[Id, L, ?]] =
-    writerTFunctor[Id, L]
+  def writerTIdFunctor[L]: Functor[WriterT[Id, L, ?]] = writerTIdCoflatMap
```

Comment:
We should be able to delete this now that `writerTIdCoflatMap` exists. You might want to move it up to where this was.